src/lib/eslint/JavascriptEslintStrategy.ts

```diff
@@ -7,17 +7,23 @@ import path = require('path');
 import {ESLint} from 'eslint';
 
 const ES_CONFIG: ESLint.Options = {
-	"baseConfig": {},
 	"overrideConfig": {
+		"parser": "@babel/eslint-parser",
 		"parserOptions": {
-			"sourceType": "module",
-			"ecmaVersion": 2018,
+			"requireConfigFile": false,
+			"babelOptions": {
+				"parserOpts": {
+					"plugins": ["classProperties", ["decorators", {"decoratorsBeforeExport": false}]]
```

Comment:
@gkjung Could you help me understand the use case for these plugins?

Reply:
I believe it's what allows the parser to understand the lwc syntax because without it, we still get the parsing errors. I tried it because it's used in LWCEslintStrategy.

Reply:
Makes sense, thanks